www/src/views/builder/index.ejs

```diff
@@ -107,7 +107,7 @@
 								<div class="col-4 col-sm-2 text-center">
 									<label class="col-form-label col-form-label-sm" for="spiInput">Spi</label>
 									<input type="number" id="spiInput" class="form-control form-control-sm" ng-model="selectedList.baseStats.spirit" ng-change="onStatChanged()" ng-model-options="{debounce:250}">
-									<span ng-show="showStatQuests()">13&nbsp;<i class="far fa-question-circle" data-toggle="tooltip" data-placement="bottom" data-container=".multiline-tooltip" data-html="true" title="<div><span>...has learned of the art and spirit of music.</span></div>" lh-tooltip></i></span>
+									<span ng-show="showStatQuests()">3&nbsp;<i class="far fa-question-circle" data-toggle="tooltip" data-placement="bottom" data-container=".multiline-tooltip" data-html="true" title="<div><span>...has learned of the art and spirit of music.</span><br><span>...traveled to the Well of Saegais and ate from the magical hazelnut trees.</span></div>" lh-tooltip></i></span>
```

Comment:
Good find! The 13 was left in when the hazelnut stuff was added.

Unfortunately, you also put the whois for hazelnut back into the tooltip, can you remove this?